generate-sources/use-plugin/src/main/kotlin/io/arrowkt/example/AnnotatedFunctionInjection.kt

```diff
@@ -0,0 +1,16 @@
+package io.arrowkt.example
+
+class AnnotatedFunctionInjection {
+
+    @MetaLog
+    fun doStuff(input: Int): Int {
+        return input * 2
+    }
+}
+
+fun main() {
+    // Assert logging is invoked
+    val testInstance = AnnotatedFunctionInjection()
+    val result = testInstance.doStuff(5)
+    println("result: $result")
+}
```

Comment:
Would be great to test somehow else, but this was the easiest way I know at the moment